spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
My intuition was close I think. Given the following:

```elisp
(use-package foo
  :init
  'foo-init
  :config
  'foo-config)
```
We get the following macro-expansion (tidied up a bit for clarity)

```elisp
(progn
  'foo-init
  (if (not (require 'foo nil t))
      (message "Could not load package %s" "foo")
    'foo-config))
```

Somehow I ran into a corner case when debugging my startup process where this package in particular wasn't getting autoloaded, so the call to `volatile-highlights-mode` would throw an error. Aggressively requiring the feature fixes this.